db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
+doc = Nokogiri::HTML(open("http://www.spanishdict.com/blog/6/top-100-spanish-words-people-really-want-to-know"))
+
+doc.xpath('//tr')[1..100].each do |x|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.